pkg/roachpb/api.proto

```diff
@@ -2753,6 +2753,7 @@ message TenantSetting {
 // tenant, which directly factors into their bill.
 message TenantConsumption {
   double r_u = 1;
+  double k_v_r_u = 8;
```

Comment:
Name this `kv_r_u`. If this is to generate the right looking Go code field, we can use a `[(gogoproto.customname) = "KVRU"]` annotation.

Reply:
Done.